kopf/engines/logging.py

```diff
@@ -39,9 +40,10 @@ def createLock(self):
     def filter(self, record):
         # Only those which have a k8s object referred (see: `ObjectLogger`).
         # Otherwise, we have nothing to post, and nothing to do.
+        fit_lvl = record.levelno >= config.EventsConfig.events_loglevel
```

Comment:
fit?

Reply:
maybe filter_level instead? :)

Reply:
"Fits the minimal level condition". Some help with proper wording is appreciated.

Reply:
ooh

Reply:
loglevel_higher? 🤔

Reply:
@samurang87 Fixed (probably).